src/views/Home.vue

```diff
@@ -11,12 +11,26 @@
 			<div style="height: 1em"/>
 
 			<h3>Jul 24-25, 2021 • Virtual Event</h3>
-			<!-- <input 
-				type="text"
-				placeholder="Enter your email for updates!"
-				v-model="email"
-				@keyup.enter="addMailList"
-			> -->
+
+			<!-- Begin Mailchimp Signup Form -->
+			<form action="https://cutc.us10.list-manage.com/subscribe/post?u=73d566326ae6bb629b42fb5fd&amp;id=e28a4a51fe" method="post" 
+					target="_blank" novalidate>
+
+				<transition name="fade" mode="out-in">
+					<div v-if="signup">
+						<input type="email" v-model="email" placeholder="Enter your email for updates!" required>
+						<!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
+						<div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_73d566326ae6bb629b42fb5fd_e28a4a51fe" tabindex="-1" value=""></div>
+						<input type="submit" v-on:click="submitEmail">
```

Comment:
Could you specify  `value="Submit"` here? Certain browsers display different default text for the submit button, such as Firefox doing this:
![image](https://user-images.githubusercontent.com/5107795/105104358-10da7c80-5a80-11eb-9151-40a878cbf9f8.png)